components/style/properties/properties.mako.rs

```diff
@@ -1091,6 +1098,10 @@ impl PropertyDeclaration {
                     match input.try(|i| CSSWideKeyword::parse(context, i)) {
                         Ok(CSSWideKeyword::InheritKeyword) => {
                             % for sub_property in shorthand.sub_properties:
+                                if properties_seen.get_${property.ident}() {
```

Comment:
Also, I believe I've made a mistake - I'm pretty sure the line you commented on (and some lines below) should have a `sub_property.ident` instead of `property.ident`. Is this correct?
That brings me to another question (kinda off topic): should this thing even compile? I read the `property.ident` value outside the `property` loop, and yet the `property` is still available. Is mako supposed to allow this behaviour?